### PR TITLE
Valid countries

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

**Comment by nicpottier** [comment]
Looks good norbert.

**Comment by norkans7** [comment]
This is ready for review @nicpottier

File: ureport/countries/views.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import pycountry
+from smartmin.views import SmartCRUDL, SmartListView
+from .models import CountryAlias
+
+class CountryALiasCRUDL(SmartCRUDL):
```
Comment (nicpottier) [inline ureport/countries/views.py]: Typo with case in ALias.

**Comment by nicpottier** [comment]
Looks good Norbert, were you able to find a projection that had slightly less focus on Greenland? Would be nice to have that as I'm sure it will come up.

File: ureport/countries/views.py (hunk 1 above)
Comment (norkans7) [inline ureport/countries/views.py]: Okay I fix that

**Comment by norkans7** [comment]
for the projection I used http://jsfiddle.net/gh/get/jquery/1.11.0/highslide-software/highcharts.com/tree/master/samples/mapdata/custom/world-eckert3

**Comment by nicpottier** [comment]
Nice, perfect.

**Pull request merged**